bulk/bulk.go

```diff
@@ -37,11 +51,48 @@ func (r *Resource) CreateJob(options Options) (*Job, error) {
 	return job, nil
 }
 
-// AllJobs will retrieve all of the bulk 2.0 jobs.
-func (r *Resource) AllJobs(parameters Parameters) (*Jobs, error) {
-	jobs, err := newJobs(r.session, parameters)
+// JobsInfo will retrieve all of the bulk 2.0 jobs.
+func (r *Resource) JobsInfo(parameters Parameters) ([]Response, error) {
```

Comment:
This is an API change.  This means that any person that wants to use Query 2.0 and Bulk 2.0 will have to change their code if it uses the old API.  I would like to avoid that at all cost and see if there is another way to accomplish this.

Reply:
I agree. But, it was confusing  to me the *newJobs* and and *Jobs* is not actually Jobs, is just information about Jobs. I just change to be more intuitive functions.
I will think another way to accomplish this.